src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerSearchViewModel.cs

```diff
@@ -1564,12 +1573,20 @@ public void DisableSearchTextBox()
         /// <summary>
         /// Clear after closing down
         /// </summary>
-        internal void Close()
+        internal void PackageManagerViewClose()
+        {
+            SearchAndUpdateResults(String.Empty); // reset the search text property
+            InitialResultsLoaded = false;
+            TimedOut = false;
+        }
+
+        /// <summary>
+        /// Remove PackageManagerSearchViewModel resources
+        /// </summary>
+        internal void Dispose()
         {
-            TimedOut = false;   // reset the timedout screen 
-            InitialResultsLoaded = false;   // reset the loading screen settings
-            RequestShowFileDialog -= OnRequestShowFileDialog;
```

Comment:
Is this still unsubscribed somewhere?

Reply:
Actually, no. This line ` ` never made sense in a way, because the `RequestShowFileDialog` is null here. 

We are subscribing each individual `PackageManagerSearchElementViewModel` to `OnRequestShowFileDialog` here https://github.com/DynamoDS/Dynamo/blob/c64fe07eb2fd68f0ab86ebc143a4c3ffa4afc4c2/src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerSearchViewModel.cs#L1363. Speaking with @reddyashish these should automatically unsubscribe when the collection is destroyed.